motion_profiles.py

```diff
@@ -0,0 +1,107 @@
+"""Provide motion profiles for smooth and efficient motion"""
+
+from typing import NamedTuple
+
+
+class MotionProfile(NamedTuple):
+    """Motion profile representing a specific desired move"""
+
+    acceleration: float
+    deceleration: float
+    acceleration_time: float
+    deceleration_time: float
+    max_speed: float
+    end_time: float
```

Comment:
What are these for? Aren't these variables used in method scope rather than class scope?